### PR TITLE
Make dissoc idempotent per #258.

### PR DESCRIPTION
**Pull request by machinelearningdeveloper**

### CODE REVIEW

File: toolz/dicttoolz.py [hunk 1]
```diff
@@ -208,10 +208,13 @@ def dissoc(d, *keys):
     {'x': 1}
     >>> dissoc({'x': 1, 'y': 2}, 'y', 'x')
     {}
+    >>> dissoc({'x': 1}, 'x') == dissoc(dissoc({'x': 1}, 'x'), 'x')
```
Comment (mrocklin) [inline toolz/dicttoolz.py]: Perhaps 

```python
>>> dissoc({'x': 1}, 'y')  # Ignores missing keys
{'x': 1}
```

**Comment by mrocklin** [comment]
@eigenhombre I know that you're for this.  @eriknw any objections/thoughts?

**Comment by eriknw** [comment]
+1

**Comment by tayloramurphy** [comment]
+1

Is this going to be merged soon?

**Comment by tayloramurphy** [comment]
Awesome! Thanks @eriknw

**Pull request merged**